nsxt/policy_utils.go

```diff
@@ -188,6 +188,55 @@ func getResourceIDFromResourcePath(rPath string, rType string) string {
 	return ""
 }
 
+func parseStandardPolicyPath(path string) ([]string, error) {
```

Comment:
Do you just need to collect the list of identifiers in the policy path, or will it also matter figuring out what is what?

For instance in:
/orgs/myorg/projects/myproj/infra/tier-1s/mygw1"
parent[2] is a router id

but in:
/orgs/myorg/projects/myproj/vpcs/myvpc/tier-1s/mygw1
parent[2] is a VPC id

Reply:
Parent list will be used as parameters for API calls, so in case of orgs/projects/vpc the ids will be used as parameters in API call in the same way as gw id or segment id would be used.